people-ops/international-compensation.html.md

```diff
@@ -0,0 +1,75 @@
+---
+title: International compensation
+meta_description: >
+  Nebulab strives to hire the best fit-talents both in culture and skill-set. Our international
+  compensation policy aims to pay the right salary for them. Find out how we handle this!
+---
+
+> Note: The work here is inspired by [Gitlab's approach to international compensation](https://about.gitlab.com/handbook/total-rewards/compensation/#paying-local-rates),
+> which we believed to be a good starting point.
+
+We hire people in different regions and countries. Every market is different, and it can be
+challenging to be competitive and fair towards other team members in making an offer for the same
```

Comment:
I think you can remove this `towards other team members in`. It's pretty clear even without it and will make the sentence easier to read.

Reply:
good one!